File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/LoteDetalleResponse.java
package com.farmacia.sanidadbackend.dto;

import lombok.Data;
import java.math.BigDecimal; // <-- Importar

@Data
public class LoteDetalleResponse {
    private Long id;
    private Long medicamentoId;
    private String medicamentoNombre;
    private String medicamentoPresentacion;
    private String fabricante;
    private Integer cantidad;
    private BigDecimal precioUnitario; // <-- NUEVO
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/LoteDetalleResponse.kt
package com.sanidad.movil.data.remote.dto

data class LoteDetalleResponse(
    val id: Long,
    val medicamentoId: Long,
    val medicamentoNombre: String,
    val medicamentoPresentacion: String,
    val fabricante: String,
    val cantidad: Int,
    val precioUnitario: Double
)